pychunkedgraph/graph/cache.py

```diff
@@ -67,17 +68,18 @@ def children_decorated(node_id):
 
         return children_decorated(node_id)
 
-    def atomic_cross_edges(self, node_id):
-        @cached(cache=self.atomic_cx_edges_cache, key=lambda node_id: node_id)
-        def atomic_cross_edges_decorated(node_id):
-            edges = self._cg.get_atomic_cross_edges(
-                np.array([node_id], dtype=NODE_ID), raw_only=True
+    def cross_chunk_edges(self, node_id, *, time_stamp: datetime = None):
+        @cached(cache=self.cross_chunk_edges_cache, key=lambda node_id: node_id)
+        def cross_edges_decorated(node_id):
+            edges = self._cg.get_cross_chunk_edges(
+                np.array([node_id], dtype=NODE_ID), raw_only=True, time_stamp=time_stamp
             )
             return edges[node_id]
 
-        return atomic_cross_edges_decorated(node_id)
+        return cross_edges_decorated(node_id)
 
     def parents_multiple(self, node_ids: np.ndarray, *, time_stamp: datetime = None):
+        node_ids = np.array(node_ids, dtype=NODE_ID)
```

Comment:
Just saw this here - same as in #458: `np.array` will by default create a copy. `np.asarray` will avoid copies, if the requirements are already met.